calicoctl/calico_ctl/node.py

```diff
@@ -612,8 +612,16 @@ def node_remove(remove_endpoints):
                         "in the normal way.")
         sys.exit(1)
 
+    # Remove the veths, and release all IPs associated with the endpoint.
+    ips = set()
     for endpoint in endpoints:
         remove_veth(endpoint.name)
+        ips |= {net.ip for net in endpoint.ipv4_nets}
```

Comment:
Urm, it's a perfectly common operator.  It's completely standard Python.

If it was `blah += [1, 2, 3]` should I comment what the operator is doing?